config/locales/en/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  notifiers:
+    course:
+      assessment:
+        answer:
+          comment_notifier:
+            replied:
+              user_notifications:
+                email:
+                  subject: '%{course} New comment on %{topic}'
+                  message: >
+                    %{post_author} commented, %{post}
```

Comment:
I think `post.text` is rich text and stored in html format -> `"-1\n\nAgain the function definition is incorrect. You need to compare the real part with real part and the imaginary part with imaginary part."`

I think you need to call `format_html(post.text)`, but probably at the notification partial. I'm not too sure myself, better if you can test posting a rich text post and taking a look at the email.